### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ishaan2053**
### Adding a New Icon? 
Nope 🙂

### Adding a New Feature or fixing a bug? 
<p> <strong>Polishing readme.md:</strong></p>
<li>Corrected a lot of grammatical errors in the official readme.</li>
<li>Added appropriate punctuation to the readme.</li>

### CODE REVIEW

**Review by Snailedlt — APPROVED** [review]
Neat! Looks good to me :heavy_check_mark: 
Thank you for the contribution! :rocket:

**Comment by Ishaan2053** [comment]
Thank you 🫡
Always happy to contribute🙂

File: README.md [hunk 1]
```diff
@@ -67,20 +67,22 @@
...
-    Thank you to our contributors and the <a href="https://icomoon.io/#home">IcoMoon app</a>. Devicon would not be possible without you.
-</p>
 
 
-<h2 id="getting-started">Getting started</h2>
+<h2 id="getting-started">Getting Started</h2>
 <p>
     For a super fast setup go check <a href="https://devicon.dev">devicon.dev</a>.<br />
-    You can either <a href="#getting-started-svg">use the raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is 
+    You can either use the<a href="#getting-started-svg"> raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is
```
Comment (Panquesito7) [inline README.md:85]: Extra space in the link.
```suggestion
    You can either use the<a href="#getting-started-svg">raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is 
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
I guess we should merge this into `develop`, rather than `master`.

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Like @Panquesito7 already mentioned, this PR needs to be merged into `develop` instead of `master`. Please rebase on `develop` and change the base branch :)

**Comment by Ishaan2053** [comment]
I've changed the base branch from `master` to `develop`. I hope we're good to go now.

**Review by Panquesito7 — DISMISSED** [review]
Awesome. Thanks! 🚀

File: README.md [hunk 2]
```diff
@@ -67,20 +67,22 @@
...
-    Thank you to our contributors and the <a href="https://icomoon.io/#home">IcoMoon app</a>. Devicon would not be possible without you.
-</p>
 
 
-<h2 id="getting-started">Getting started</h2>
+<h2 id="getting-started">Getting Started</h2>
 <p>
     For a super fast setup go check <a href="https://devicon.dev">devicon.dev</a>.<br />
-    You can either <a href="#getting-started-svg">use the raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is 
+    You can either use the<a href="#getting-started-svg">raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is
```
Comment (Snailedlt) [inline README.md:85]:
```suggestion
    You can either use the <a href="#getting-started-svg">raw SVG</a> icons or our <a href="#getting-started-font">devicon font</a> (which is 
```

Comment (Snailedlt) [reply README.md:85]: Missing space between words

**Review by Snailedlt — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -167,8 +169,8 @@ Add CSS rules in your stylesheet.
 
 <h2 id="discord-community">Discord community</h2>
 <p>
-We have a Discord community for Devicons. You can easily request icons, discuss, and have a good time talking with the community members! Join today: https://discord.gg/hScy8KWACQ.
-<b>Note that the Discord server is unofficial, and Devicons is still being maintained via GitHub.</b>
+We have a Discord community for Devicons. You can easily request icons, discuss, and have a good time talking with the community members! Join today: https://discord.gg/hScy8KWACQ.</p>
+<p><b>Note that the Discord server is unofficial, and Devicons is still being maintained via GitHub.</b>
```
Comment (Panquesito7) [inline README.md:173]: Relevant, but for another discussion/issue: is the Discord server still unofficial? I think we should state it as official now.

**Review by Panquesito7 — APPROVED** [review]
Overall, looking good. Thanks! 🚀

File: README.md (hunk 3 above)
Comment (Snailedlt) [reply README.md:173]: Whoops, didn't see this comment before merging, but yes... I agree. The Discord server should be considered official

Comment (Ishaan2053) [reply README.md:173]: #1641 PR fixes this🙂, mentioning the server as official.

**Pull request merged**